service/src/main/java/uk/gov/hmcts/reform/fpl/config/CtscEmailLookupConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class CtscEmailLookupConfiguration {
+    private final String email;
+
+    public CtscEmailLookupConfiguration(@Value("${fpl.ctsc-inbox}") String email) {
```

Comment:
I would double check this is added everywhere it needs to be. Need to open a pr to cnp flux and stuff I believe. I can't remember exactly where, maybe see catch up with Etim / Piotr who have done something similar recently.

Reply:
Caught up with Etim and Tome and added where the variable is needed.